packages/fdc3-get-agent/test/step-definitions/intent-resolver.steps.ts

```diff
@@ -4,7 +4,7 @@ import { handleResolve } from "@kite9/testing";
 import { DefaultDesktopAgentIntentResolver } from "../../src/ui/DefaultDesktopAgentIntentResolver";
 import { INTENT_RESPOLVER_URL } from "../support/MockFDC3Server";
 
-
+// TODO: Replace 'any' with useful typings
 const contextMap: Record<string, any> = {
```

Comment:
The `Context` type would be the obvious one 
```suggestion
import { Context } from "@kite9/fdc3-context";

const contextMap: Record<string, Context> = {
```